### ISSUE TITLE
Icon request: file-check, file-x, folder-check, folder-x

### ISSUE THREAD

**Issue by controversial** [feathericons/feather#529]
## Icon Request

* Icon name: “check” and “x” variants of the `file` and `folder` icons
* Use case: There are already `file-plus` and `file-minus` icons (and the equivalent for folders) to symbolize adding and removing these objects, but being able to symbolize success and validation in relation to files and folders would be enormously useful for many apps, especially those involving uploads
* Screenshots of similar icons:
<img width="243" alt="screen shot 2019-02-04 at 8 43 37 pm" src="https://user-images.githubusercontent.com/10377391/52248691-9a0d3e80-28bd-11e9-81cc-50f35c451375.png">
<img width="246" alt="screen shot 2019-02-04 at 8 43 26 pm" src="https://user-images.githubusercontent.com/10377391/52248693-9bd70200-28bd-11e9-8b29-8d75480a6227.png">
(all from [unicons](https://iconscout.com/unicons))

**Comment by johnletey** [comment]
@colebemis @locness3 @controversial How does [this](https://www.figma.com/file/vMeF1zmyWdW68RxzQsYeCj/file-folder-check-times) look?

**Comment by locness3** [comment]
> @colebemis @locness3 @controversial How does [this](https://www.figma.com/file/vMeF1zmyWdW68RxzQsYeCj/file-folder-check-times) look?

This one looks great, but make sure to be in compliance with #171 .

**Comment by johnletey** [comment]
> This one looks great, but make sure to be in compliance with #171 .

Yup ... looking like it is in compliance ... going to submit a PR!

**Comment by locness3** [comment]
@johnlestey
No they are not.

**Comment by johnletey** [comment]
> No they are not.

@locness3 What is wrong?

**Comment by locness3** [comment]
Strokes = Borders. This means you should use borders instead of fills. Also, you shouldn't use the masks provided for Figma when submitting your icons. You should use only shapes themselves.

**Comment by johnletey** [comment]
Refactoring now ... will update you to have a look when I'm done!

**Comment by johnletey** [comment]
@locness3 I've updated the icons ... please take a look! They now use SVG shapes instead of components and also I have shrunk the "x" to have the same top and bottom space as the "check"

**Comment by locness3** [comment]
> @locness3 I've updated the icons ... please take a look! They now use SVG shapes instead of components and also I have shrunk the "x" to have the same top and bottom space as the "check"

Great, but please use a border for the checks and X, instead of a fill. BTW, you should remove the original "file" and "folder" groups from the icons.

**Comment by johnletey** [comment]
@locness3 The checks and Xs now have a 1px border (is that a violation of #171?) and I have removed all icon groups ... solely vectors now!

**Comment by locness3** [comment]
> a 1px border (is that a violation of #171?)

#171 is again very clear : "Every line and shape has a 2px center-aligned stroke with round joins and round caps"

**Comment by johnletey** [comment]
@locness3 Have changed ... but IMHO, they look ugly now

**Comment by johnletey** [comment]
@locness3 @controversial How does it look now?

<img width="477" alt="Screen Shot 2019-07-02 at 7 31 27 AM" src="https://user-images.githubusercontent.com/30328854/60482745-777c2780-9c9b-11e9-8254-17adb22fed35.png">

**Comment by locness3** [comment]
Maybe make the checks and crosses smaller.
![Screenshot_2019-07-02 Figma(1)](https://user-images.githubusercontent.com/37651007/60485413-32f18b80-9c9d-11e9-818a-2de6420e7d3a.png)

**Comment by johnletey** [comment]
@locness3 @ahtohbi4 How does it look now? [Figma](https://www.figma.com/file/C7cEHvYfSsGpvx9utbnFnhJy/Feather-Icons?node-id=0%3A1)

<img width="606" alt="Screen Shot 2019-07-08 at 8 00 15 AM" src="https://user-images.githubusercontent.com/30328854/60784192-a2acbe00-a156-11e9-8d71-72adfceef10a.png">

**Comment by locness3** [comment]
Perfect!
